docs/Server/API/V1/relay.md

```diff
@@ -0,0 +1,143 @@
+# `POST /api/v1/relay/<uri>`
+
+This API creates a dataset resource by reading data from a Relay server in two
+steps. The provided URI represents a "manifest file" stored on the Relay server,
+a JSON file (`application/json` MIME format) defining the original tarball name,
+the MD5 hash value, optional metadata key/value pairs for the dataset, and a
+URI from which the Pbench Server expects to `GET` an `application/octet-stream`
+of a file created by `tar` and compressed with the `xz` program.
+
+Primarily this is expected to be a native Pbench Agent tarball with a specific
+structure; however with the `server.archiveonly` metadata key the Pbench Server
+can be used to archive and manage metadata for any tarball passed through a
+Relay server.
```

Comment:
Given the paragraph at line 10, we should probably tone-down lines 7-8.  I _think_ the only requirement (in lines 7-8's paragraph) is that the URI refer to the result file which the Pbench Server should `GET`.

We can then go on to explain at line 10 that it is expected to have been produced by the Pbench Agent (and, therefore, should be an `xz`-compressed `tar` output) but that the Server can accept an arbitrary file if the manefest specifies `server.archiveonly`, although the Server won't attempt to unpack or perform any analysis on it.